server/tomlconfig/verify.go

```diff
@@ -132,6 +134,28 @@ func (c *SXGCertConfig) verify() error {
 	return errs.ErrorOrNil()
 }
 
+func (c *SXGACMEConfig) verify() error {
+	var errs *multierror.Error
+
+	if !c.EnableCertAutoRenew {
+		return errs.ErrorOrNil()
+	}
+
+	if c.DiscoveryURL == "" {
```

Comment:
Also verify it's a valid URL with the https scheme?

NB: There's `verifyURL` below but it probably doesn't suit this case.

Reply:
Done.  I think verifyURL is ok to use here.

Reply:
Is it okay to accept a relative url (an absolute path) though?

Reply:
Currently, it's not because only Digicert supplies this URL (and they don't use webpackager).

Reply:
Sorry, I'm confused.  Why isn't verifyURL good enough? I thought it already checked that it was absolute path. Is that not the case?

Reply:
`verifyURL` treats absolute urls as _valid_; we don't want it for `DiscoveryURL`.

Reply:
To clarify: absolute URL format is: "protocol://domain/path", relative URL format is: "path/X/Y/Z".
verifyURL accepts absolute URL format, rejects relative URL format.
DiscoveryURL should follow absolute URL format.
Did I misunderstand what you meant?

Reply:
Sorry, I made a critical typo in my comment. Here's the correct version:
`verifyURL` treats absolute **paths** (relative urls starting with a slash) as valid, and we don't want it for `DiscoveryURL`.
Does this make sense now?